README.md

```diff
@@ -89,174 +127,120 @@ pipeline.execute() // execute the pipeline
-     |{
-     |  "pipeline" : [
-     |    {
-     |      "filename" : "/path/to/las",
-     |      "type" : "readers.las"
-     |    },
-     |    {
-     |      "type" : "filters.crop"
-     |    },
-     |    {
-     |      "filename" : "/path/to/new/las",
-     |      "type" : "writers.las"
-     |    }
-     |  ]
-     |}
-  """.stripMargin
-  
+"""
+|{
+|  "pipeline" : [
+|    {
+|      "filename" : "/path/to/las",
+|      "type" : "readers.las"
+|    },
+|    {
+|      "type" : "filters.crop"
+|    },
+|    {
+|      "filename" : "/path/to/new/las",
+|      "type" : "writers.las"
+|    }
+|  ]
+|}
+""".stripMargin
 // The same, but using scala DSL
 val pc = ReadLas("/path/to/las") ~ FilterCrop() ~ WriteLas("/path/to/new/las")
-
 // The same, but using RawExpr, to support not implemented PDAL Pipeline API features
 // RawExpr accepts a circe.Json type, which can be a json object of any desired complexity
-val pcWithRawExpr = ReadLas("/path/to/las") ~ RawExpr(Map("type" -> "filters.crop").asJson) ~ WriteLas("/path/to/new/las") 
-
+val pcWithRawExpr = ReadLas("/path/to/las") ~ RawExpr(Map("type" -> "filters.crop").asJson) ~ WriteLas("/path/to/new/las")
 // Create Pipelines from the constructed expressions
 val pipelinePc = pc.toPipeline
 val pipelinePc = pcWithRawExpr.toPipline
 ```
 
-## How to compile
-
-Development purposes (including binaries) compilation:
-  1. Install PDAL (using brew / package managers (unix) / build from sources / etc)     
-  2. Build native libs `./sbt native/nativeCompile` (optionally, binaries would be built during tests run)
-  3. Run `./sbt core/test` to run PDAL tests
-
-Only Java development purposes compilation:
-  1. Provide `$LD_LIBRARY_PATH` or `$DYLD_LIBRARY_PATH`
-  2. If you don't want to provide global variable you can pass `-Djava.library.path=<path>` into sbt:
-    `./sbt -Djava.library.path=<path>`
-  3. Set `PDAL_DEPEND_ON_NATIVE=false` (to disable `native` project build)
-  4. Run `PDAL_DEPEND_ON_NATIVE=false ./sbt`
-
-Finally the possible command to launch and build PDAL JNI bindings could be:
-
-```bash
-# Including binaries build
-./sbt
+## Known Issues
+1. In case of not installed as global PDAL change [this](./java/native/src/CMakeLists.txt#L25) line to:
+```cmake
+set(CMAKE_CXX_FLAGS "$ENV{PDAL_LD_FLAGS} $ENV{PDAL_CXX_FLAGS} -std=c++11")
 ```
-
+In this case sbt launch would be the following:
 ```bash
-# Java side development without binaries build
-PDAL_DEPEND_ON_NATIVE=false ./sbt -Djava.library.path=<path>
+PDAL_LD_FLAGS=`pdal-config --libs` PDAL_CXX_FLAGS=`pdal-config --includes` ./sbt
 ```
-
-### Possible issues and solutions
-
-1. In case of not installed as global PDAL change [this](./java/native/src/CMakeLists.txt#L25) line to:
-
-  ```cmake
-  set(CMAKE_CXX_FLAGS "$ENV{PDAL_LD_FLAGS} $ENV{PDAL_CXX_FLAGS} -std=c++11")
-  ```
-  In this case sbt launch would be the following:
-
-  ```bash
-  PDAL_LD_FLAGS=`pdal-config --libs` PDAL_CXX_FLAGS=`pdal-config --includes` ./sbt
-  ```
```

Comment:
why this is gone? or that's again a bad fmt / identation.

Hmm really this PR did really smth with raw md file identation / newlines.